pkg/cloud/azure/storagebillingparser.go

```diff
@@ -184,3 +211,38 @@ func (asbp *AzureStorageBillingParser) timeToMonthString(input time.Time) string
 	endOfMonth := input.AddDate(0, 1, -input.Day())
 	return startOfMonth.Format(format) + "-" + endOfMonth.Format(format)
 }
+
+// deleteFilesOlderThan7d recursively walks the directory specified and deletes
+// files which have not been modified in the last 7 days. Returns a list of
+// files deleted.
+func (asbp *AzureStorageBillingParser) deleteFilesOlderThan7d(localPath string) ([]string, error) {
```

Comment:
This doesn't quite feel right to me, but I think it is fine for now. If we do end up supporting partitioned exports doing it this way will make thing easier.

Reply:
Agree. Deleting files based off an arbitrary time cutoff is weird.

The goal though, was to make sure that local disk space doesn't fill up too quickly. With CloudCost running, we would eventually be downloading every Azure month-to-date bill to disk. We should clean up the files that we're likely no longer using?